types/index.d.ts

```diff
@@ -0,0 +1,282 @@
+            * @param n value to increment
+         */
+        increment(key: string, n: number): void;
+
+        /**
+         * Decrement a counter by N.
+         * @param key Key
+         * @param n value to decrement
+         */
+        decrement(key: string, n: number): void;
+
+        /**
+         * Start a new stack with the given name and with a given max length. In the example below, we start the "notices" stack that will be capped at 20 items at all times. You do not have to call .startStack() to start pushing values to a stack — if you pushed to a non-existing stack, the stack would automatically be generated and its length would be capped at 15 items by default.
+         * @param name Name of the stack
+         * @param length Lenght of the stack
+         */
+        startStack(name: string, length: number): void;
+
+        /**
+         * Push a new value to a stack. A stack can be pre-configured using .startStack() but does not have to be. If .stack() is called without .startStack(), the default length of the stack will be 15 items.
+         * @param name Name of the stack
+         * @param value Value
+         */
+        stack(name: string, value: string): void;
+
+        /**
+         * Remove a stack and any of its values.
+         * @param name Name of the stack
+         */
+        stopStack(name: string): void;
+
+        /**
+         * Set a specific value to the values key-value object in the status page. Useful for, for example, connection status indications (e.g. whether a certain connection is "CONNECTED" or "CLOSED", etc).
+         * @param name Name of the key
+         * @param value Value of the key
+         */
+        set(name: string, value: any): void;
+
+        /**
+         * Un-set a specific key within the values block.
+         * @param name Name of the key
+         */
+        unset(name: string): void;
+
+        /**
+         * Increment a throughput counter with the given name. The throughput will get automatically calculated per second, per minute and per hour, and will appear in throughput object on the status page. Any new names will trigger automatic creation of the given throughput counter.
+         * @param name Name of the throughput counter.
+         */
+        throughput(name: string): void;
+
+        /**
+         * Increment a throughput counter with the given name. The throughput will get automatically calculated per second, per minute and per hour, and will appear in throughput object on the status page. Any new names will trigger automatic creation of the given throughput counter.
+         * @param name Name of the throughput counter.
+         */
+        clearThroughput(name: string): void;
+
+        /**
+         * Return process status information (memory, os, network, etc...)
+         * @param request
+         */
+        generateStatus(request?: ServerRequest): StatusInfo;
+    }
+
+    /**
+     * Kardia Status
+     */
+    interface Status {
```

Comment:
is there any possibility to show that `Status` actually _extends/merges_ from `Worker` ? they have lot in common